README.adoc

```diff
@@ -3,21 +3,59 @@
+- Container deployments to Kubernetes
+- Load balancers
+- Security groups
+- Server groups
+- Clusters
+- Firewalls
+- Functions
+
+
+Facilitate GitOps, and grant developers control of provisioning infrastructure for apps. Natively, use Spinnaker’s cloud driver to deploy to all of the major cloud providers, and the Kubernetes V2 provider for Kubernetes deployments. You may even orchestrate configuration and firmware changes as part of Spinnaker delivery pipelines.
+
+= Why Do I Need Spinnaker?
+
+With Spinnaker, create a “paved road” for application delivery, with guardrails that ensure only valid infrastructure and configuration reach production.
+Free development teams from burdensome ops provisioning while automating reinforcement of business and regulatory requirements. Delivery automation
+strategies such as canary deployments provide the safety necessary to capture value from quick innovation, while protecting against business and end-user
+ impact. As the most mature and widely productionalized continuous delivery platform, Spinnaker can apply the expertise of Netflix, Google, Microsoft,
+ and Amazon to your SDLC. User companies include Target, Salesforce, Airbnb, Cerner, Adobe, and JPMorgan Chase.
+ 
+= Tech Specs
+
+Spinnaker consists of a number of https://www.spinnaker.io/reference/architecture/[independent microservices], with the Halyard CLI administration service
+managing the lifecycle of the other services. A variety of SDLC tools integrate with Spinnaker, and its plugin framework offers an alternative to extending
+and maintaining individual Spinnaker extensions. Plugins allow us to add system integrations without restarting or updating Spinnaker, broadening its potential
+to easily leverage the entire software delivery toolchain. With this in place, Spinnaker is evolving towards leaner binaries and a smaller threat surface,
+enabling performance and operational advantages. Meanwhile, managed delivery, a newer Spinnaker featureset, provides declarative definitions of common
+infrastructure and other requirements; users can declare requirements using those
+prebuilt definitions and move changes through environments via a visual interface.
+
+'''
+
+This repository contains ancillary configuration, build, and testing tools for the Spinnaker project. It also centralizes issue tracking across Spinnaker, accepting issues from across the project.  
+The core code making up Spinnaker’s microservices is found in the other https://github.com/spinnaker[Spinnaker repositories].
 
 = Using Spinnaker
 
-If you are only interested in using Spinnaker, please refer to the main
+Spinnaker users should refer to the main
 Spinnaker site and https://www.spinnaker.io/setup/[Installation] guide.
 
-If you want more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Documentation Overview].
+For more information on how Spinnaker is designed, see the https://www.spinnaker.io/concepts/[Documentation Overview].
```

Comment:
Is this a new or preferred way of creating markdown links?

Reply:
The readme for this repo is written in ASCII doc. Not sure why it is, but this is the typical way to do it in ASCII doc

Reply:
lol I forgot about this while I was working on edits based on @ttomsu 's comments and had to fix them after  : \